benchmarking/commons/benchmark_definitions/lstm_wikitext2.py

```diff
@@ -18,16 +18,20 @@
     METRIC_NAME,
     RESOURCE_ATTR,
 )
+from syne_tune.backend.sagemaker_backend.estimators import (
+    DEFAULT_GPU_INSTANCE_SMALL,
```

Comment:
Better names would be DEFAULT_GPU_INSTANCE_1GPU and DEFAULT_GPU_INSTANCE_4GPU

Reply:
Ok, makes sense. Changed